src/model/timestamp.ts

```diff
@@ -76,9 +76,30 @@ export class Timestamp {
   }
 
   public dump(): string {
-    return `${this.epochMs},${this.parts.join(",")}`;
+    const partsStr = this.parts.length > 0 ? "," + this.parts.join(",") : ""
+    return `${this.epochMs},${partsStr}`;
   }
-}
+
+  public toString(): string {
+    return this.dump()
+  }
+
+  public static parse(rawTimestamp: string): Timestamp {
+      const timestampParts = rawTimestamp
+      .replace("(", "")
```

Comment:
A more reliable way would be to use a regex for parsing. Right now something like `12)34.()96` would be treated as a valid timestamp, but it would be better to throw an error on encountering such strings. We can leave it as is for now, it's not that important